test/npmignore.js

```diff
@@ -0,0 +1,71 @@
+import path from 'path';
+import test from 'ava';
+import mockery from 'mockery';
+import sinon from 'sinon';
+
+let moduleUnderTest;
+
+const gitUtilApi = {
+	async newFilesSinceLastRelease() {
```

Comment:
```suggestion
	newFilesSinceLastRelease: async () => {}
```

Same for `pkgDirApi`.